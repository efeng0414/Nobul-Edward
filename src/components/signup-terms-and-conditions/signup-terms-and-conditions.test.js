import React from "react";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";

import TermsAndConditions from "./";

describe("Terms And Conditions", () => {
  const containerProps = {
    intl: global.getIntl()
  };
  const containerJSX = (
    <IntlProvider {...containerProps}>
      <TermsAndConditions />
    </IntlProvider>
  );
  describe("Container Snapshot", () => {
    const snapshotJSX = (
      <IntlProvider {...global.intl}>{containerJSX}</IntlProvider>
    );
    const containerSnapshot = renderer.create(snapshotJSX).toJSON();
    it("should match stored snapshot", () => {
      expect(containerSnapshot).toMatchSnapshot();
    });
  });
});
