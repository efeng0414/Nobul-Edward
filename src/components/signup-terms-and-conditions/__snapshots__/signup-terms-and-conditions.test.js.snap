// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms And Conditions Container Snapshot should match stored snapshot 1`] = `
<div>
  By checking this box you confirm that you have read and agree to the 
  <span
    className="accept-terms-link"
  >
    <a
      href="https://corp.nobul.com/legal/terms-and-conditions/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Terms of Service
    </a>
  </span>
   and that you have read our Privacy Policy.
</div>
`;
